slack/web/classes/blocks.py

```diff
@@ -0,0 +1,189 @@
+from typing import List, Set, Union
+
+from .elements import BlockElement, InteractiveElement
+from .objects import (
+    JsonObject,
+    JsonValidator,
+    MarkdownTextObject,
+    PlainTextObject,
+    TextObject,
+    extract_json,
+)
+
+
+class Block(JsonObject):
```

Comment:
We had talked about a `BlockBuilder` or something along those lines, but when I started prototyping one the usage was pretty awkward - there's just too many combinations of blocks + elements + objects for a builder to really make sense, IMO